### DISCUSSION TITLE
Scan deletes current stations list

### DISCUSSION THREAD

**Discussion by theJazzman42** [KejPi/AbracaDABra#81] [General]
I'd appreciate if I could choose keeping my stations list or not. Of course changes ought to be made. 

But I have some multiplexes I receive some days and other days I don't.

Would be nice to be able to keep them as proof and a kind of "badge of honor".

**Comment by KejPi** [comment]
This feature could be very confusing for regular users. I mean it is pretty common that once you start the scan, you loose the service list. I could think about that but only as hidden feature. 
BTW: You do not need to run full scan every time, you can just manually select the channel in expert mode and if there is any signal in the channel, it will be added to the list.

**Reply by theJazzman42** [reply]
> BTW: You do not need to run full scan every time, you can just manually select the channel in expert mode and if there is any signal in the channel, it will be added to the list.

Sure. But it was annoying today when I was able to receive 18 (!!) different multiplexes clicking all channels separately...

**Comment by KejPi** [comment]
It will be less next time ;-) 
But OK, juts for you I will add it ;-)

**Comment by KejPi** [comment]
Implemented in 5a8997741dc899d76d8c4bc53ac5cc88e162de9e as hidden option. You can enable this feature by editing INI file and setting `keepServiceListOnScan` option to true:

```
keepServiceListOnScan=true
```

**Comment by theJazzman42** [comment]
thx

**Comment by theJazzman42** [comment]
Build the package according to the instructions. Still get a warning, a scan will delete all previous results. And it does.

**Comment by KejPi** [comment]
Have you enabled the feature in INI file? It is disabled by default because it would confuse regular users.

**Comment by theJazzman42** [comment]
Yes, I did because you told me to.

**Reply by KejPi** [reply]
Then you did some mistake somewhere. The latest version from repository does not show any message when the feature is enabled, in fact the band scan starts automatically without asking.

**Comment by theJazzman42** [comment]
Moved the *.ini to a different spot, so a new one was created. Added the option and now it works.

Any plans to release a new Windows version soon?

**Reply by KejPi** [reply]
You do not have to add the option, it is created automatically with false value after first run of the application.
I will probably release new version during July, I still need to finish something and I will not have time to work on that in next 2 weeks.
